core-web/apps/dotcms-ui/src/app/api/services/dot-router/dot-router.service.ts

```diff
@@ -228,8 +228,8 @@ export class DotRouterService {
         ]);
     }
 
-    goToURL(url: string): void {
-        this.router.navigate([url]);
+    goToURL(url: string, extras?: NavigationExtras): void {
+        this.router.navigate([url], extras);
```

Comment:
What happen if we don't pass anything `router.navigate` handles the `undefined`?